public/static/admin/config.yml

```diff
@@ -23,7 +23,7 @@ collections:
           widget: "list"
           fields:
             - {label: "Title", name: "title", widget: "string"}
-            - {label: "Publish Date", name: "date", widget: "date", format: "YYYY-MM-DD"}
+            - {label: "Publish Date", name: "date", widget: "datetime", format: "YYYY-MM-DD", timeFormat: false, pickerUtc: true}
```

Comment:
See [my Netlify CMS PR](https://github.com/netlify/netlify-cms/pull/3721) for too much detail on why we need to set `pickerUtc: true`. 😅